components/intake.py

```diff
@@ -56,21 +57,18 @@ def infraredDistance(self):
         infrared_voltage = self.infrared.getVoltage()
         """Makes sure that the voltage is above 0.00001"""
         voltage = max(infrared_voltage, 0.00001)
-        distance = 12.84*math.pow(voltage, -0.9824)
-        """This makes sure that the distance is above 4.5 and below 35"""
+        distance = 12.84 * voltage ** -0.9824
+        """This makes sure that the distance is above 4.5cm and below 35cm"""
         self.cube_distance = max(min(distance, 35.0), 4.5)
```

Comment:
So, uh, can we please stop reinventing the wheel here? The RobotPy common driver library exists for a reason.